src/modules/item/export.ts

```diff
@@ -176,6 +176,15 @@ function buildItemEntityMetadata(collection: Collection, item: Item): StandardCa
     catalystItem.emoteDataV0 = {
       loop: (item.data as EmoteData).category === EmoteCategory.LOOP
     }
+    // add missing properties from wearable schema so catalyst wont reject the deployment
+    catalystItem.data.category = WearableCategory.HAT
```

Comment:
I used the `hat` category randomly, it's not used by the explorer aparently, any wearable category would work, but leaving the `simple` or `loop` values would make the schema reject it.